aten/src/ATen/native/Copy.cpp

```diff
@@ -112,21 +114,21 @@ void _copy_same_type__cpu(Tensor& self, const Tensor& src) {
     } else if (copy_transpose_valid(self, src)) {
       _copy_same_type_transpose_(self, src);
     } else {
-#ifdef _OPENMP
-      if (!in_parallel_region()) {
-        AT_DISPATCH_ALL_TYPES_AND(
-          at::ScalarType::Half, self.scalar_type(), "_copy_same_type_", [&]() {
-            at::CPU_tensor_parallel_apply2<scalar_t, scalar_t>(
-                self, src, [](scalar_t& self_val, const scalar_t& src_val) {
-                  self_val = src_val;
-                });
-          });
-      } else {
-        serial_path = true;
-      }
-#else
+// #ifdef _OPENMP
```

Comment:
This needs to be renabled before it lands.

Reply:
Something isn't working well together with the sorting of strides and coalescing that happens in the building of the TensorIterator.